### PR TITLE
TIMESHEETS!

### PR DESCRIPTION
**Pull request by KINFOO**
Hoping picked dates does it.

### CODE REVIEW

File: scripts/timesheet.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+        var username = user.name;
+        airbot.logger.debug("Reminding timesheet to:", username);
+        if (_.contains(blacklist, username)) {
+            airbot.logger.debug("Nothing to remind to:", username);
+            return;
+        }
+
+        var greeting = [
+            "Hi *<@" + username + ">*,",
```
Comment (sebz) [inline scripts/timesheet.js]: `*` are not interpreted

File: scripts/timesheet.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+/**
+ * Description:
+ *     Reminds everyone to fill their timesheets
+ */
+
+var _ = require("lodash");
+var schedule = require("node-schedule");
+
+var GIF_URL = "https://media.giphy.com/media/QNv4mKyspa8j6/giphy.gif";
```
Comment (sebz) [inline scripts/timesheet.js]: Why not extracting it in the `.airbotrc`?

**Comment by sebz** [comment]
Two remarks, otherwise LGTM :+1:

File: scripts/timesheet.js (hunk 1 above)
Comment (KINFOO) [inline scripts/timesheet.js]: Trap

Comment (KINFOO) [inline scripts/timesheet.js]: When I tested with Air2D2, code was:

    "Hi*<@" + username + ">*,",

Like you, I saw it, then added an extra `" "` after `Hi`, to get current:

    "Hi *<@" + username + ">*,",

It is fine now.

Comment (sebz) [inline scripts/timesheet.js]: Oh OK :smile: , my bad!

**Comment by KINFOO** [comment]
@sebz `AIRBOT_TIMESHEET_GIF_URL` is in `.airbotrc.template`

**Comment by sebz** [comment]
LPTM :metal:

**Pull request merged**